syntax = "proto3";

message GenericMessage {
  oneof value {
    int32 int_value = 1;
    float float_value = 2;
    string string_value = 3;
    bytes bytes_value = 4;  // 支持二进制数据
    bool bool_value = 5;
    double double_value = 6;
    int64 int64_value = 7;
  }
}
